Description: Make OQGraph compile with Boost 1.6
 Boost 1.6 changed the interface in boost::graph. Make sure to not use
 the now removed defines.
 .
 This patch should be removed when upstream provides a fix.
Author: Vicen»õiu Ciorbaru <cvicentiu@gmail.com>
Bug: https://jira.mariadb.org/browse/MDEV-9479

--- mariadb-10.0-10.0.25.orig/storage/oqgraph/oqgraph_shim.h
+++ mariadb-10.0-10.0.25/storage/oqgraph/oqgraph_shim.h
@@ -254,7 +254,7 @@ namespace boost
     typedef no_property type;
   };
 
-#if BOOST_VERSION >= 104601
+#if BOOST_VERSION >= 104601 && BOOST_VERSION < 106000
   template <>
   struct graph_bundle_type<oqgraph3::graph>
   {
